source/_includes/common-tasks/backups_container.md

```diff
@@ -0,0 +1,26 @@
+## Backups
+
+Backup of your Home Assistant. They are stored in a compressed archive file (.tar). Backups are made from {% my supervisor_backups title="Settings > System > Backups" %}. There is also a service available that allows you to trigger the creation of a backup from an automation. Backups are stored in the `/backup` directory.
+
+A full backup includes the following directories:
+
+* `config`
+* `share`
+* `ssl`
+* `media`
+
+A partial backup consists of any number of the above default directories.
+
+{% include common-tasks/backups_create.md %}
+
+### Restoring a Backup
+
+You can make use of backup which you have copied off of a previous install. Ensure Home Assistant is not running.
+
+Untar the backup file and `homeassistant.tar.gz` inside the backup file and copy the contents of `homeassistant/data` folder to the `/config` volume.
+
+If you're looking to slim down your backup, check if your configuration directory contains a large database file (`home-assistant_v2.db`). See the [`recorder`](/integrations/recorder/) integration page for options to keep your database data down to a size that won't cause issues. Note the keep days, purge interval, and include/exclude options.
+
+Start Home Assistant container.
+
+{% include common-tasks/backups_offsite.md %}
```

Comment:
These do not apply? The ones listed in that snippet are add-ons, add-ons are not available on container installation types.